src/main/java/seedu/momentum/model/VersionedProjectBook.java

```diff
@@ -57,7 +58,7 @@ public void commit(ViewMode viewMode, Project currentProject, Predicate<TrackedI
     public void undo() {
         assert canUndoCommand();
         shiftPointer(UNDO);
-        ReadOnlyProjectBook undoVersion = projectBookStateList.get(currentStatePointer);
+        ProjectBookWithUi undoVersion = projectBookStateList.get(currentStatePointer);
```

Comment:
Why is this being changed?

Reply:
Oh forgot to change it back. Changed it to make an earlier implementation work.